docs/integrations/sources/file.md

```diff
@@ -0,0 +1,98 @@
+# Files
+
+## Overview
+
+File are often exchanged or published in various remote locations. This source aims to support an expanding range of file formats and storage providers. The File source supports Full Refresh syncs. That is, every time a sync is run, Airbyte will copy all rows in the file and columns you set up for replication into the destination in a new table.
+
+### Output schema
+
+At this time, this source produces only a single stream for the target file as it replicates only one file at a time for the moment. We'll be considering to improve this behavior by globing folders or use patterns to capture more files in the next iterations as well as more file formats and storage providers.
+Note that you should provide the `dataset_name` which dictates how the table will be identified in the destination (since `URL` can be made of complex characters)
+
+### Features
+
+| Feature | Supported? |
+| :--- | :--- |
+| Full Refresh Sync | Yes |
+| Incremental Sync | No |
+| Replicate Incremental Deletes | No |
+| Replicate Folders \(multiple Files\) | No |
+| Replicate Glob Patterns \(multiple Files\) | No |
+
+### Storage Providers
+
+| Storage Providers | Supported? |
+| :--- | :--- |
+| HTTPS | Yes |
+| Google Cloud Storage | Yes |
+| Amazon Web Services S3 | Yes |
+| SFTP / SSH / SCP | Yes |
+| WebHDFS | Untested |
```

Comment:
if we haven't tested them we shouldn't release them and keep them hidden.

Reply:
I haven't tested it because i didnt plan on installing hadoop...

It's being tested here: https://github.com/RaRe-Technologies/smart_open/blob/develop/integration-tests/test_webhdfs.py
but not as part of airbyte and the extra code we put around is not that much in this case

But sure we can hide it